src/Atmos/Model/AtmosModel.jl

```diff
@@ -27,7 +27,12 @@ using ..TurbulenceClosures
 import ..TurbulenceClosures: turbulence_tensors
 using ..TurbulenceConvection
 
-import ..Thermodynamics: internal_energy, total_specific_enthalpy
+import ..Thermodynamics: 
```

Comment:
```suggestion
using ..Thermodynamics: 
```

`import` should be used when we want to define new methods for the same function